rules/ruleset.go

```diff
@@ -52,6 +54,7 @@ var (
 	errNoRuleSnapshots      = errors.New("rule has no snapshots")
 	ruleActionErrorFmt      = "cannot %s rule %s"
 	ruleSetActionErrorFmt   = "cannot %s ruleset %s"
+	missingOpInChanges      = "changes must contain an op"
```

Comment:
just define the error here?

Reply:
I think the comment is asking you to define `errMissingOpInChanges = ...`.

Reply:
O yeah. Thats a good call.